src/apportioned-emissions/annual/annual-unit-data.repository.ts

```diff
@@ -127,6 +127,28 @@ export class AnnualUnitDataRepository extends Repository<AnnualUnitDataView> {
     return results;
   }
 
+  async getEmissionsNationalAggregation(
+    req: Request,
+    params: PaginatedAnnualApportionedEmissionsParamsDTO,
+  ): Promise<AnnualUnitDataView[]> {
+    let totalCount: number;
+    let results: AnnualUnitDataView[];
+    const { page, perPage } = params;
+
+    const selectColumns = ['aud.year'],
+      orderByColumns = ['aud.year'];
```

Comment:
const orderByColumns = ['aud.year'];

Reply:
The above is basically the same because of the comma after line 138 but I'll make the changes anyway so its easier to see.

Reply:
Oh my bad I missed the comma. Either way is fine